packages/analyticsdx-template-lint/test/unit/validator.test.ts

```diff
@@ -86,7 +86,8 @@ describe('FileTemplateValidator', () => {
       missingField: 'releaseInfo',
       lintWarning: {
         range: { start: { line: 0, character: 0 }, end: { line: 0, character: 0 } },
-        message: 'App templates must have at least 1 dashboard, dataflow, externalFile, lens, or recipe specified',
+        message:
+          'App templates must have at least 1 component, dataflow, dataset, dataTransform, dashboard, externalFile, lens, predictions, recipe, or story specified',
```

Comment:
Total nit, sorry but it caught my eye: precisions is plural whereas everything else is singular. Same thing with the message at line 100

Reply:
Good eye. I'll fix that up.